---
layout: home
title: fan
# articles:
#   excerpt_type: html
---
<div class="hero">
  <h1>WHO WE ARE </h1>
  <p>
    We are a computational lab in the Informatics Institute at the Heersink School of Medicine, University of Alabama at Birmingham. We conduct cutting-edge bioinformatics and genomics research with a focus on new sequencing platforms, including PacBio, Nanopore, 10X Genomics, etc.
  </p>
</div>

<div class="homepage-container">

  <!-- Section À propos -->
  <section class="project-summary">
    <h2>Vision of the lab</h2>
    <p>
     The advances of sequencing and other biotechnologies are revolutionizing our understanding of biology to an unprecedented extent. Our lab aims to take full advantage of these technologies to explore meaningful biological insights from our genomes, both healthy and unhealthy ones. The ultimate goal is to improve the health of every individual with our expertise in Bioinformatics.
    </p>
    <p>
    </p>
    <h2>Develop algorithms for new sequencing data analysis</h2>
    <p>
     Although there are many tools available for next-generation sequencing data analysis, the vast volumes of sequencing data that are being generated at a decreasing cost still require more sensitive and specific algorithms to analyze in an efficient manner. The third-generation sequencing (TGS) platforms (PacBio, Oxford nanopore, etc.) can generate reads of thousands of base pairs, which can cover a large proportion of the genome in a haplotype. However, the error rates of the TGS platforms are very high (e.g. ~15% for PacBio), which is extremely challenging for data analysis. But there are many open questions for Bioinformaticians to develop novel methods to overcome the drawbacks and fully exploit the properties of the TGS data. Besides, there are newly developed biotechniques, such as 10X Genomics, strand-seq, improved single-cell sequencing, etc., which are generating miscellaneous sequencing data types. We are also interested in providing efficient solutions to deal with these types of data.
    </p>
    <p>
    </p>
    <h2>Functions and mechanisms of genomic rearrangements</h2>
    <p>
     Compared to single nucleotide variants and short indels, genomic rearrangements or structural variations (SVs) can contribute to more polymorphisms within species and more divergence between species. However, it is still an open question in terms of the functions and mechanisms of SVs. For example, the observation and hypothesis of chromothripsis are still mysterious. We are interested in an in-depth analysis of SVs in a hypothesis-driven manner.
    </p>
    <p>
    </p>
    <h2>Disease genomics</h2>
    <p>
    The disease is an abnormal phenotype. There is a long-lasting complex problem regarding how the genotype (mixing with epigenetic alterations and modifications and environment) determines the phenotype. We are interested in a wide range of questions, for example, discovering the causing variants, pan-cancer/disease genomics data analysis, integrated data analysis, etc.
    </p>
    <p>
    </p>
    <h2>Collaborations</h2>
    <p>
    We have extensive collaborations around the UAB campus and nationwide.
    </p>
    
    <p>
    </p>

    <h4>Recent News</h4>
    <p>
      News 1
    </p>

<!-- Section Derniers articles -->
<div class="hero">
  <h2>Lately Published</h2>
</div>
